src/index.html

```diff
@@ -1,13 +1,41 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img src="images/imac.jpeg" alt="iMac" class="card__image">
+      </div>
+      <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+      <p class="card__code">Код товара: 195434</p>
+      <div class="card__rating">
+        <div class="block-stars">
+          <div class="block-stars__star"></div>
+          <div class="block-stars__star"></div>
+          <div class="block-stars__star"></div>
+          <div class="block-stars__star"></div>
+          <div class="block-stars__star"></div>
+        </div>
+        <div class="card__review">
+          <p class="review">Отзывов: 5</p>
+        </div>
+      </div>
+      <div class="card__price price">
+        <p class="price__text">Цена:</p>
+        <p class="price__sum">69 999 грн</p>
+      </div>
+      <div class="card__button" data-qa="hover">
+        <a class="button" href="#">Купить</a>
+      </div>
```

Comment:
```suggestion
      <a class="button" href="#">Купить</a>
```
Зараз неправильно працює hover ефект. просто стилізуй лінку, як кнопку, і пропиши hover для всієї кнопки

https://user-images.githubusercontent.com/66570142/159685133-46d9a113-1582-4625-8e37-fe2c10077922.mov